Cinema index

julianageller opened the pull request.

marcusaandahl commented:
@julianageller @Rasmus-Melberg  you guys got some conflicts in your cinema show
And the maps for cinemas for the master (ramsus' branch pull request where map was added) are not showing up when the cinema is pressed

julianageller commented:
I fixed it - it was just because Rasmus added the map div to the cinema index and I added the sessions. Right now we have both on the same page, but ideally we'd create a new page and route for the map, as agreed earlier today. I can do that tomorrow morning.
